src/main/kotlin/ru/hh/android/synthetic_plugin/delegates/ViewBindingPropertyDelegate.kt

```diff
@@ -75,43 +107,96 @@ class ViewBindingPropertyDelegate(
         classes.forEach { (psiClass, ktClass) ->
             val parents = ClassParentsFinder(psiClass)
             when {
+                parents.isChildOf(ANDROID_ACTIVITY_CLASS) -> processActivity(ktClass)
                 parents.isChildOf(ANDROID_FRAGMENT_CLASS) -> processFragment(ktClass)
                 parents.isChildOf(ANDROID_VIEW_CLASS) -> processView(ktClass)
                 parents.isChildOf(HH_CELL_INTERFACE) -> processCell(ktClass)
                 else -> println("Can't add ViewBinding property to class: ${psiClass.qualifiedName}")
             }
-            addImports(bindingQualifiedClassName)
+            addImports(*getBindingQualifiedClassNames().toTypedArray())
+            formatCode(ktClass)
+        }
+    }
+
+    private fun processActivity(ktClass: KtClass) {
+        if (isUsingViewBindingPropertyDelegate) {
```

Comment:
The `Strategy` pattern looks like the appropriate thing to split different implementations. 
https://refactoring.guru/design-patterns/strategy

What do you think?

Reply:
Sure, it's a good idea, i will implement it!

Reply:
Split on base and variable implementation, but not quite sure about naming